force-app/main/default/classes/PlatformEventMonitoringController.cls

```diff
@@ -1,12 +1,21 @@
 /**
 * Custom Controller to show Platform Events Subscriber Status
 **/ 
-public class PlatformEventMonitoringController {
+public with sharing class PlatformEventMonitoringController {
     
     @testVisible
     public static List<EventBusSubscriber> subscriptionInfo {
-        get {return [SELECT Topic, ExternalId, Name, Position, Retries, Status, Tip, Type, LastError 
-                     FROM EventBusSubscriber order by Topic, Name LIMIT 100]; }
+        get {
+            Boolean hasCustomPermission = FeatureManagement.checkPermission('Monitor_Platform_Events');
+            if (hasCustomPermission && isSafeObject('EventBusSubscriber')) {
+                return [SELECT Topic, ExternalId, Name, Position, Retries, Status, Tip, Type, LastError 
+                     FROM EventBusSubscriber order by Topic, Name LIMIT 100]; 
+                     }
```

Comment:
Definitely like the custom permission check @MaurizioBella , a tiny nit, I would make that literal a constant.
Also, regarding the isSafeObject, feels like overkill because you cannot give OLS (or FLS) on EventBusSubscriber.

Reply:
yes, I do agree. thanks @bobbywhitesfdc